Lib/test/libregrtest/__init__.py

```diff
@@ -0,0 +1,5 @@
+# We import importlib *ASAP* in order to test #15386
+import importlib
+
```

Comment:
Would it make sense to place the `libregrtest` folder outside the `Lib` folder? Or has this discussion already happened?